examples/jms-service-consume-message/jms_service_consume_message.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on consuming messages from a JMS provider in an asynchronous manner using Ballerina.
```

Comment:
```suggestion
description: BBE on consuming messages from a JMS provider in an asynchronous manner using Ballerina.
```
Ditto

Reply:
Related to: https://github.com/ballerina-platform/ballerina-distribution/pull/4880#discussion_r1323928811